### PR TITLE
feat(pager): add optional page change event

### PR DESCRIPTION
**Pull request by louislewis2**
add an optional page change event, which will be called when the page value changes. this will simplify use cases where the pager is used stand alone or when server side paging is required. update readme to reflect the bindable property

### CODE REVIEW

File: src/pager.js [hunk 1]
```diff
@@ -38,6 +39,9 @@ export class Pager {
   pageChanged(newValue, oldValue) {
     if (newValue !== oldValue) {
       this.goToPage(newValue);
+	  if (this.onPageChanged !== null) {
```
Comment (RWOverdijk) [inline src/pager.js]: `if (typeof this.onPageChanged === 'function') {}` is probably a better idea.

Also, add two spaces in front of the if (indentation is wrong now) and add a newline before the if.

But reading your issue, I'm not sure this solves your problem. You might want to wait for it to finish loading data. If not, this PR is fine :)

**Review by RWOverdijk — CHANGES_REQUESTED** [review]
Almost perfect!

**Comment by RWOverdijk** [comment]
I'm sorry to bother you, but indentation is still wrong, mainly because you're using tabs (we use spaces). You also left out the typeof check. It should look like this:

```js
  pageChanged(newValue, oldValue) {
    if (newValue !== oldValue) {
      this.goToPage(newValue);

      if (typeof this.onPageChanged === 'function') {
        this.onPageChanged(newValue, oldValue);
      }
    }
  }
```

**Comment by louislewis2** [comment]
Sorry, battling with an editor. currently using notepad to edit, when I open the source in visual studio code, it messes up all the files with styling.. Let me try correct it again ;)

**Comment by RWOverdijk** [comment]
No problem :) Thanks for contributing!

**Comment by louislewis2** [comment]
😨 Did it come right this time around? 😣

**Comment by RWOverdijk** [comment]
Mmmm no haha. You can just use my snippet of code. I'd do it, but you should have your name on this code :)

**Comment by louislewis2** [comment]
What editor are you using? Maybe I can download the same one and just get it right...

**Comment by louislewis2** [comment]
Okay, I tried once again..... 😁 
This time I disabled the format on save and turned on view white space.
With a little bit of luck it might be closer to the truth this time.

**Comment by RWOverdijk** [comment]
It's not, sorry. You changed the formatting to 4 spaces instead of 2

**Comment by louislewis2** [comment]
If that one is not right, then I don't know.
However I do think that adding in a editor config file for visual studio code can maybe save some future trouble?

**Comment by RWOverdijk** [comment]
You did it! :D I'll release this

**Review by RWOverdijk — APPROVED** [review]

**Comment by louislewis2** [comment]
Thanks for your patience on this one. It is much appreciated. I'm certain next time will be much smoother ;)

**Comment by RWOverdijk** [comment]
Published as 0.2.0

Low version for something this stable makes me happy :)

**Pull request merged**